### ISSUE TITLE
Setting basePath to demo environment does not work

### ISSUE THREAD

**Issue by niebuhrFirstcon** [docusign/docusign-click-csharp-client#10]
https://github.com/docusign/docusign-click-csharp-client/blob/df074b2d6716f1a9a0655e08e2d8f0f84cd9a940/sdk/src/DocuSign.Click/Client/DocuSignClient.cs#L108

This line states, that my basePath will only be used, if it is null or empty. Or do i miss something?

**Comment by InbarGazit** [comment]
No, this lines means we default to production, if you do not set anything. We set it to what you have in the configuration object, but if you have nothing (null or empty) we default to Production_REST_BasePath

**Comment by niebuhrFirstcon** [comment]
Isn't there a ! missing in front of the string.IsNullOrEmpty? I don't get it... I cloned the repo and set the !, now it works for me...

**Comment by kenharris** [comment]
Thanks for your report @niebuhrFirstcon - this does indeed look like a bug. Our team will look into it.
